src/cache.rs

```diff
@@ -138,7 +134,7 @@ pub fn set<P: AsRef<Path>>(
     };
     if let Err(e) = write_sync(
         &settings.cache_dir,
-        cache_key(path, settings, autofix),
+        cache_key(path, settings),
```

Comment:
Ah, actually, need to revert this part. We _do_ have to include the autofix setting here. Otherwise, if we run with autofix disabled, then enabled, we'll never pick up the fixes.